sonarts-core/docs/rules/no-useless-cast.md

```diff
@@ -0,0 +1,36 @@
+# no-useless-cast
+
+The TypeScript compiler automatically casts a variable to the relevant type inside conditionals where it is possible to infer (usage of `typeof`, `instanceof` etc). These compiler feature makes casts and not-null assertions unnecessary.
```

Comment:
@vilchik-elena  use singular "this compiler feature"

Reply:
fixed